snippets/visualbasic/VS_Snippets_CLR_System/system.io.directoryinfo.enumeratedirectories/vb/program.vb

```diff
@@ -1,19 +1,19 @@
 '<Snippet1>
-Imports System
 Imports System.IO
 
 Class Program
     Public Shared Sub Main(ByVal args As String())
-        Dim diTop As New DirectoryInfo("d:\")
+        Dim dirPath As String = Environment.GetFolderPath(Environment.SpecialFolder.MyDocuments)
+        Dim diTop As New DirectoryInfo(dirPath)
         Try
             For Each fi In diTop.EnumerateFiles()
                 Try
                     ' Display each file over 10 MB;
                     If fi.Length > 10000000 Then
                         Console.WriteLine("{0}" & vbTab & vbTab & "{1}", fi.FullName, fi.Length.ToString("N0"))
                     End If
-                Catch UnAuthTop As UnauthorizedAccessException
-                    Console.WriteLine("{0}", UnAuthTop.Message)
+                Catch unAuthTop As UnauthorizedAccessException
```

Comment:
This is a nit, but typically exception variables aren't given meaningful names -- they're just named `e` or `ex`.